// Copyright 2010, 2011 Yunrang Inc. All Rights Reserved.
// Author: jianxu@yunrang.com (Xu Jian)

package util_mysql;

message TestMysqlInfo {
  optional int64 data_int64 = 1;
  optional int32 data_int32 = 2;
  optional uint64 data_uint64 = 3;
  optional uint32 data_uint32 = 4;
  optional double data_double = 5;
}
